### ISSUE TITLE
[sift] Could it be possible to support RGBA images?

### ISSUE THREAD

**Issue by vallsv** [silx-kit/silx#1841]
Is that something easy or not?

**Comment by kif** [comment]
On Tue, 22 May 2018 09:59:32 +0000 (UTC)
Valentin Valls <notifications@github.com> wrote:

> Is that something easy or not?

We did it. (I do it instensively in my timelaps image alignment)
Basically it pre-processes the color image and renders it in black&white before using sift.

If it does not work, it is a bug.

-- 
Jérôme Kieffer
tel +33 476 882 445

**Comment by vallsv** [comment]
Yes i see it is working well on RGB. But maybe you misunderstand me, i am talking about RGB+alpha.

**Comment by kif** [comment]
On Tue, 22 May 2018 04:27:34 -0700
Valentin Valls <notifications@github.com> wrote:

> Yes i see it is working well on RGB. But maybe you misunderstand me, i am talking about RGB+alpha.

Extending the code from RGB to RGBA shouldn't be difficult ...
